src/main/java/com/uber/kafkaSpraynozzle/KafkaSpraynozzle.java

```diff
@@ -66,40 +67,21 @@ public static void main(String[] args) throws Exception {
         }
 
         if (spraynozzleHA) {
+            buffering = true;
```

Comment:
Synced up offline. Good point, different spraynozzles will want to be HA with different reliability semantics.